server/container/guice/mailbox/pom.xml

```diff
@@ -40,6 +40,10 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>apache-james-mailbox-event-memory</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-mailbox-store</artifactId>
```

Comment:
The need for this additional dependency mitigates the gain of not repeating the MailboxAnnotationListener bindings.
`apache-james-mailbox-store` is an evil module as it has no clear definition (I mean, it could be called apache-james-mailbox-commons which is a hint it's bad)